### PR TITLE
Repl mobile

### PR DESCRIPTION
**Pull request by diksipav**

### CODE REVIEW

File: shared/common/ui/navtabs/mobileNavTabs.module.scss
Comment (jaclarke) [inline shared/common/ui/navtabs/mobileNavTabs.module.scss:1]: I don't think these changes are meant to be here? Looks like they're undoing one of the commits on main?

File: shared/inspector/index.tsx [hunk 1]
```diff
@@ -137,19 +137,21 @@ const RowList = observer(function RowList({
 
   if (disableVirtualisedRendering) {
     const rows = maxHeight ? items.slice(0, maxHeight + 1) : items;
-    const showExpandedButton =
+    const showExpandBtn =
       maxHeight &&
       (rows.length > maxHeight ||
         rows.reduce((s, r) => s + (r.height ?? 1), 0) > maxHeight);
 
+    setShowExpandBtn(!!showExpandBtn);
```
Comment (jaclarke) [inline shared/inspector/index.tsx:145]: You can't call a state update directly inside the render function like this. Why does the 'show more' button need to move?

**Comment by jaclarke** [comment]
We need a run button on mobile. It's kind of tricky to do `ctrl+enter` on mobile. We need a run button, and I think the empty prompt should probably be taller to make it easier to tap on: 

image

File: shared/common/ui/navtabs/mobileNavTabs.module.scss
Comment (diksipav) [reply shared/common/ui/navtabs/mobileNavTabs.module.scss:1]: yes, that's what I asked you on Slack, I had conflicts and wasn't sure if I resolved them correctly.

**Review by jaclarke — APPROVED** [review]

**Pull request merged**